ui/raidboss/data/07-dt/trial/zoraal-ja-ex.txt

```diff
@@ -0,0 +1,160 @@
+11.2 "--sync--" StartsUsing { id: "9398", source: "Zoraal Ja" } window 11.2,4
+16.0 "Actualize" Ability { id: "9398", source: "Zoraal Ja" }
+26.3 "Multidirectional Divide (lines drop)" Ability { id: "93A2", source: "Zoraal Ja" }
+37.3 "Multidirectional Divide (damage)" Ability { id: "93A3", source: "Zoraal Ja" }
+38.4 "Half Full" Ability { id: "9380|939E", source: "Zoraal Ja" }
+38.4 "Forward Edge/Backward Edge" #Ability { id: "9972|937F", source: "Zoraal Ja" }
+47.7 "Multidirectional Divide (lines drop)" Ability { id: "93A2", source: "Zoraal Ja" }
+58.7 "Multidirectional Divide (damage)" Ability { id: "93A3", source: "Zoraal Ja" }
+58.9 "Regicidal Rage" Ability { id: "993C", source: "Zoraal Ja" }
+
+# Swords & Tiles Phase 1
+74.2 "Dawn of an Age" Ability { id: "9397", source: "Zoraal Ja" }
+88.3 "Vollok" Ability { id: "9357", source: "Zoraal Ja" }
+98.6 "Sync" Ability { id: "9359", source: "Zoraal Ja" }
+107.6 "Chasm of Vollok" Ability { id: "939A", source: "Zoraal Ja" }
+108.2 "Half Full" Ability { id: "9380|939E", source: "Zoraal Ja" }
+117.1 "Greater Gateway" Ability { id: "9381", source: "Zoraal Ja" }
+125.2 "Blade Warp" Ability { id: "935E", source: "Zoraal Ja" }
+133.3 "Forged Track (cast)" Ability { id: "935F", source: "Zoraal Ja" }
+141.6 "Fiery Edge/Stormy Edge" Ability { id: "9382|9385", source: "Fang" }
+141.6 "Forged Track (damage)" #Ability { id: "939D", source: "Fang" }
+146.4 "Chasm of Vollok" Ability { id: "9389", source: "Zoraal Ja" }
+151.4 "Actualize" Ability { id: "9398", source: "Zoraal Ja" } window 5,5
+
+# Lines, Lines, Lines Phase 1
+163.7 "Projection of Triumph" Ability { id: "938A", source: "Zoraal Ja" }
+# Don't sync 'Siege/Walls of Vollok' due to spamminess
+173.8 "Siege of Vollok/Walls of Vollok" #Ability { id: "938B|938C", source: "Fang" }
+# Second cast, with identical ID, but happens with the first Siege/Walls so don't display
+#173.8 "Projection of Triumph" Ability { id: "938A", source: "Zoraal Ja" }
+178.8 "Siege of Vollok/Walls of Vollok" #Ability { id: "938B|938C", source: "Fang" }
+183.8 "Siege of Vollok/Walls of Vollok" #Ability { id: "938B|938C", source: "Fang" }
+188.9 "Siege of Vollok/Walls of Vollok" #Ability { id: "938B|938C", source: "Fang" }
+192.4 "Half Full" Ability { id: "9380|939E", source: "Zoraal Ja" }
+192.4 "Forward Edge/Backward Edge" #Ability { id: "9972|937F", source: "Zoraal Ja" }
+193.9 "Siege of Vollok/Walls of Vollok" #Ability { id: "938B", source: "Fang" }
+198.9 "Siege of Vollok/Walls of Vollok" #Ability { id: "938B", source: "Fang" }
+202.4 "Actualize" Ability { id: "9398", source: "Zoraal Ja" }
+214.5 "Projection of Turmoil" Ability { id: "9A88", source: "Zoraal Ja" }
+# Might of Vollok (938D) occurs when a player crosses the line
+# Instead of syncing (which will be random based on player resolution),
+# display a long duration while the line is moving
+221.0 "Might of Vollok" duration 40
+264.8 "Bitter Whirlwind (initial)" Ability { id: "993E", source: "Zoraal Ja" }
```

Comment:
`Bitter Whirlwind` is a three-hit tankbuster. The first hit has a cast bar and a different ID. A tank swap or a tank invuln is necessary.

Maybe this would be better as `(1)`, `(2)`, `(3)`?